test/bitstyles_phoenix/bitstyles_test.exs

```diff
@@ -4,15 +4,65 @@ defmodule BitstylesPhoenix.BitstylesTest do
   import ExUnit.CaptureIO
 
   describe "classname/2" do
-    test "version 4.0.0" do
+    test "version 5.0.0" do
       assert Regex.match?(
-               ~r/4.0.0 of bitstyles is not yet supported/,
-               capture_io(:stderr, fn -> classname("u-flex", "4.0.0") end)
+               ~r/5.0.0 of bitstyles is not yet supported/,
+               capture_io(:stderr, fn -> classname("u-flex", "5.0.0") end)
              )
     end
 
+    test "version 4.2.0" do
+      assert classname("u-border-radius-0", "4.2.0") == "u-border-radius-0"
+    end
+
+    test "version 4.1.0" do
+      assert classname("u-flex", "4.0.0") == "u-flex"
+      assert classname("u-border-radius-0", "4.0.0") == "u-round-0"
+      assert classname("u-overflow-x-auto", "4.0.0") == "u-overflow-x-auto"
+      assert classname("u-overflow-y-auto", "4.0.0") == "u-overflow-y-auto"
+      assert classname("u-bg-gray-80", "4.0.0") == "u-bg-gray-80"
+      assert classname("u-fg-warning", "4.0.0") == "u-fg-warning"
+      assert classname("u-font-medium", "4.0.0") == "u-font-medium"
+      assert classname("u-line-height-min", "4.0.0") == "u-line-height-min"
+      assert classname("u-text-right", "4.0.0") == "u-text-right"
+      assert classname("u-flex-shrink-0", "4.0.0") == "u-flex-shrink-0"
+      assert classname("u-flex-grow-1", "4.0.0") == "u-flex-grow-1"
+      assert classname("u-flex-wrap", "4.0.0") == "u-flex-wrap"
+      assert classname("u-flex-col", "4.0.0") == "u-flex-col"
+      assert classname("u-grid-cols-3", "4.0.0") == "u-grid-cols-3"
+      assert classname("u-col-span-3", "4.0.0") == "u-col-span-3"
+      assert classname("u-col-start-1", "4.0.0") == "u-col-start-1"
+    end
```

Comment:
Version `4.0.0` and `4.1.0` should behave the same way. Do we want to have tests for both or rather just for one (`4.0.0`?) ? WDYT?

Reply:
I think I'd leave the tests there. But maybe update the `4.0.0` above in the assertions to `4.1.0`

Reply:
🙈 Changed it! https://github.com/bitcrowd/bitstyles_phoenix/pull/88/commits/3b6d6089c4a6af86435452bee1edfd78aa6cc3b2